src/main/java/application/Main.java

```diff
@@ -6,15 +6,18 @@
 import javafx.scene.Scene;
 import javafx.scene.image.Image;
 import javafx.stage.Stage;
-
 import java.io.IOException;
 import java.util.Objects;
+import model.Models;
 
 public class Main extends Application {
 
     @Override
     public void start(Stage primaryStage) {
         try {
+            Models modelsInitializer = new Models();
+            modelsInitializer.initializeModels();
+
```

Comment:
Remover essa parte de inicialização dos modelos pois ela é inútil nessa parte do código.

Reply:
Ponto resolvido.